### DISCUSSION TITLE
administrator dashboard in iframe

### DISCUSSION THREAD

**Discussion by ztzozon** [LiveHelperChat/livehelperchat#2021] [Q&A]
I'm currently trying to login as administrator, where the live helper chat login is opened in an iframe... but no reaction at all. insted i get a networ answer: "Failed to load response data: No content available because this request was redirected"

has anyone made similar experience?

**Comment by remdex** [comment]
lhc denies embeding it's back office as iframe unless you modify lhc

**Reply by ztzozon** [reply]
thanks for the quick answer. are the modifications possible in the ihc configuration files, or do you mean modifying the source code?

**Reply by remdex** [reply]
source code
